### ISSUE TITLE
80 failed (97: Address family not supported by protocol)

### ISSUE THREAD

**Issue by koureni** [dockur/windows#252]
this is  **docker-compose.yml** :
```
version: "3"
services:
  windows:
    image: dockurr/windows
    container_name: windows
    devices:
      - /dev/kvm
    cap_add:
      - NET_ADMIN
    ports:
      - 8006:8006
      - 13389:3389/tcp
      - 13389:3389/udp
    environment:
      RAM_SIZE: "8G"
      CPU_CORES: "4"
      DISK_SIZE: "100G"
      VERSION: "win11"
      stop_grace_period: 2m
      restart: on-failure
```
and when i start docker, some error below, the **docker log** showing 

```
root@mail:/app/windows# docker logs windows
❯ Starting Windows for Docker v2.04...
❯ For support visit https://github.com/dockur/windows

nginx: [emerg] socket() [::]:80 failed (97: Address family not supported by protocol)
❯ ERROR: Status 1 while: nginx -e stderr (line 166/10)

```
I have another NGINX Server using port 80 for reverse proxy, but i don't understand why docker will using port 80 also?

**Comment by MannODeath** [comment]
I am experiencing the same issue. My host machine does not have an IPv6 address.

**Comment by aaronnewsome** [comment]
Is there a way to launch the container on a host that doesn't have IPv6 enabled? I've disabled IPv6 on all my hosts with GRUB_CMDLINE_LINUX="ipv6.disable=1" on all hosts.

**Comment by Bad3r** [comment]
> This should be fixed now in the latest image (v2.08), please try it out and let me know!

Thank you for the update. I am testing it in a few.
Something I noticed; why do you use the special URL params for windows iso download?
```
[+] Got latest ISO download link (valid for 24 hours): https://software.download.prss.microsoft.com/dbazure/Win11_23H2_English_x64v2.iso?t=6f0d7e11-670d-4fb5-8fad-41e5957576b5&P1=1711480680&P2=601&P3=2&P4=uQ%2fUr9KPz24B4Cd5aY8%2b5moQpAPkd5hSJ9FGRnHJ7EXONLWfHb8P%2fEzHHdImnLZX6r%2bBRyTAEBHuxJdHyJWFneU50Zk5gVf46wqFuLfbWdlArdAI0R%2bJQ0RjaeFoVpSSnVDOKK5Ske5eMZOb5c0wxbFX9RWgPpQcic%2fjVxSdz0XdN6G15woWGwJTELWn%2fnZ22Iydq7rqQmx12VO2QeKEhi634fAbt0ct5M7%2fvukfJfEdW%2b4QhGhjhPsvSGfWtDg81yRYOBfDoWPo3n94fB0fufqq%2fVrK5g2BuW2I1yxHT%2fhHB6GLR0fGxoeY4c0LNdBElZ9ksEaEZqQeAvzfyoDC
curl: (22) The requested URL returned error: 403                                                              

[!] Microsoft servers returned a failing HTTP status code!
```

the URL returns a 403 but `https://software.download.prss.microsoft.com/dbazure/Win11_23H2_English_x64v2.iso` without the special URL parameters works fine. Not sure what are they used for just thought I should mention it.

**Comment by Bad3r** [comment]
> This should be fixed now in the latest image (v2.08), please try it out and let me know!

Please ignore my previous comment as using the latest tag didn't download 2.08. I can confirm that this update resolved the issue. 

Thank you for the quick fix!

**Comment by aaronnewsome** [comment]
Thank you for removing the ipv6 requirement. I'm now able to run on hosts that don't have ipv6!
